lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -447,6 +464,11 @@ impl Client {
         if !key.down {
             flags = KeyboardFlags::RELEASE;
         }
+        let extended = key.code & 0xE000 == 0xE000;
+        if extended {
+            flags |= KeyboardFlags::EXTENDED;
+        }
```

Comment:
This is where we actually set the `EXTENDED` flag; the rest of the changes are to interface with the updated IronRDP apis.